spec/fastlane_create_xcframework_action_spec.rb

```diff
@@ -217,6 +217,20 @@
         expect(result).to eq(expected_result)
       end
 
+      it 'verifies debug_symbols method when :include_debug_symbols option was not provided' do
```

Comment:
This test is basically the same code as `it 'verifies debug_symbols method'`, but without passing in the param. And it fails on the default branch currently